src/libraries/System.Net.Http.Json/tests/FunctionalTests/TestClasses.cs

```diff
@@ -14,25 +14,37 @@ internal class Person
         public int Age { get; set; }
         public string Name { get; set; }
         public Person Parent { get; set; }
+        public string PlaceOfBirth { get; set; }
```

Comment:
Just curious, what's the reason to add another property?

Reply:
To test the camel-casing of property name when serialized through the workaround I did. All the existing have only 1 word name.